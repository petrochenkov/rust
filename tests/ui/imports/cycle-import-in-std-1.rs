//@ edition: 2018

// https://github.com/rust-lang/rust/issues/124490

use ops::{self as std};
//~^ ERROR: unresolved import `ops`
//~| ERROR cannot determine resolution for the import
use std::collections::{self as ops};

fn main() {}
