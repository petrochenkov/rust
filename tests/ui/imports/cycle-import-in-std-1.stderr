error: cannot determine resolution for the import
  --> $DIR/cycle-import-in-std-1.rs:5:11
   |
LL | use ops::{self as std};
   |           ^^^^^^^^^^^

error[E0432]: unresolved import `ops`
  --> $DIR/cycle-import-in-std-1.rs:5:11
   |
LL | use ops::{self as std};
   |           ^^^^^^^^^^^ no external crate `ops`
   |
   = help: consider importing this module instead:
           std::ops

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.
